controllers/gce/utils/utils.go

```diff
@@ -103,6 +104,14 @@ func NewNamer(clusterName string) *Namer {
 	return namer
 }
 
+// NewNamer creates a new namer with a Firewall Name
+func NewNamerWithFirewall(clusterName string, firewallName string) *Namer {
```

Comment:
NewNamer is only used in tests; perhaps we should add the firewall name there instead of creating NewNamerWithFirewall. Also, the first string type can be omitted.